scripts/tdusk.lic

```diff
@@ -181,11 +186,11 @@ def attack(number)
 				end
 				pause 0.1
 			end
-			Script.start("foreach", "in room; look item") if UserVars.tdusk[:wave_number] != 25
-			Script.start("foreach", "in room; get item; stow item") if UserVars.tdusk[:wave_number] == 25
+			Script.start("foreach", "n=/^((?!arrow|pill).)*$/ in room; look item") if UserVars.tdusk[:wave_number] != 25 && !GameObj.loot.nil?
+			Script.start("foreach", "n=/^((?!arrow|pill).)*$/ in room; get item; stow item") if UserVars.tdusk[:wave_number] == 25 && !GameObj.loot.nil?
 
 		#Sorcerer Routine for Rasko
-		elsif !(npc.status =~ /dead|gone/) && Char.name =~ /Rasko|Roiken|Hammibal|Ushakaron/

```

Comment:
Beldannon will never financially recover from this

Reply:
Added back in.